llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -59,6 +61,11 @@
 #define DEBUG_TYPE "yk-control-point"
 #define JIT_STATE_PREFIX "jit-state: "
 
+#define YK_CONTROL_POINT_MT_IDX 0
+#define YK_CONTROL_POINT_LOC_IDX 1
```

Comment:
Should we consistently include `ARG` in the constant name for argument indices?

Reply:
Agreed. Fixed in 07bc55a798db.